### ISSUE TITLE
stream buffering icon needs to be smaller and more discrete

### ISSUE THREAD

**Issue by helenvarley** [upstage-org/upstage#1220]
during our performance at the weekend, there were issues with the stream which meant that some people only ever received the buffering icon. as we had the stream large, filling the stage area, this meant that the buffering icon was also large and dominated everything. if the video was also buffering there were even two large icons. screengrabs below.

is it possible to have a much smaller and less vibrant icon? and one that does not grow in size if the stream or video is increased? a nice small black and white or grey-scaled circle type of icon would blend in better.

![Uploading Screenshot from 2024-02-25 12
![Screenshot from 2024-02-25 12:03:53](https://github.com/upstage-org/upstage/assets/3437324/c141fd34-3244-4156-b843-d2b1075f32bb)
:04:56.png…]()

**Comment by elisadcg** [comment]
Hello, yes I totally agree that's during the perfromance the big icon was very disturbing.
What about a small icon wich means "streaming live". Proposal of several icons :
![Capture d’écran du 2024-02-26 14-08-57](https://github.com/upstage-org/upstage/assets/7611913/8275667b-8318-44c6-870f-cd1093748238)

**Comment by elisadcg** [comment]
In situation :
![Capture d’écran du 2024-02-26 14-11-10](https://github.com/upstage-org/upstage/assets/7611913/491658e1-9942-4e44-9e53-2b7472f02212)

**Comment by helenvarley** [comment]
great - or even simpler? the last one but with just one circle in the centre, and the 3 lines on either side animated to indicate that something is loading?

i'm adding a photo from the performance on saturday to show how the buffering icon dominated - the projection of UpStage is on the left-hand wall, and our stream from the centre front stage area is what is missing.
![425307594_722556156652138_6577843141703629219_n](https://github.com/upstage-org/upstage/assets/3437324/077bfd49-9f7e-4e68-ab5c-e37597cd477b)

**Comment by elisadcg** [comment]
![Capture d’écran du 2024-02-27 14-14-20](https://github.com/upstage-org/upstage/assets/7611913/3c2273f4-50eb-40a1-b0ab-d77f89e3676e)
preview

**Comment by helenvarley** [comment]
yes, i like the last one too. hopefully it can animate (the outer lines coming and going) to show that something is trying to load.

**Comment by vickis** [comment]
agree the last one works best - the image is pretty pixelated but hopefully on stage there is the clear definition of the radiating lines - nice work Elise :)

27 Feb 2024, 02:11 by ***@***.***:

**Comment by helenvarley** [comment]
great - maybe a bit slower animation?

**Comment by elisadcg** [comment]
![icon-stream](https://github.com/upstage-org/upstage/assets/7611913/532d1599-9f7e-4545-a943-61a2c77be5c9)
sure!

**Comment by helenvarley** [comment]
@webdev604 can this icon replace the orange icon that appears when streams or videos are buffering? 
if possible, it should have a fixed size that is not too big (see elisa's screengrab above), so that if someone has made their webcam very large the buffering icon doesn't also display large as this disrupts the image.

**Comment by helenvarley** [comment]
nudging this ...

**Comment by helenvarley** [comment]
this is actually really hard to test but i think i have seen the old orange buffering icon which we've all agreed is a bit too big and obtrusive.

can the buffering icon be replaced with this one? 

![Image](https://github.com/user-attachments/assets/1745eb0f-6705-49e3-83f9-2a96ded57cfa)
